ang/civicase/CaseOverview.js

```diff
@@ -1,73 +1,102 @@
 (function (angular, $, _) {
   var module = angular.module('civicase');
-  // Todo: To write unit tests
+
   module.directive('civicaseCaseOverview', function () {
     return {
       restrict: 'EA',
       replace: true,
       templateUrl: '~/civicase/CaseOverview.html',
       controller: civicaseCaseOverviewController,
-      scope: {}
+      scope: {},
```

Comment:
any reason for the isolated scope?

Reply:
No reason as such. Is it considered to be a better approach not isolating the scope?

Reply:
I think there is no reason to have angular having to keep track (and digest) of yet another scope

Reply:
Valid point.But, If we don't isolate the scope.while writing the testing for the members of the scope (` $scope.showHideBreakdown`, `$scope.caseListLink` etc), we have to test them inside its parent directive spec (because then the scope members of this directive will be of its parent). This will also impact the encapsulation of code of `Case overview` and make the parent directive more complex.

I don't see the advantage of making these changes. Also, tbh I have already invested too much time on it and want to close it asap.

Please let me know if you have any doubts.